libcxx/docs/Status/Cxx17Papers.csv

```diff
@@ -26,7 +26,7 @@
 "`P0013R1 <https://wg21.link/p0013r1>`__","LWG","Logical type traits rev 2","Kona","|Complete|","3.8"
 "","","","","",""
 "`P0024R2 <https://wg21.link/P0024R2>`__","LWG","The Parallelism TS Should be Standardized","Jacksonville","|Partial|",""
-"`P0226R1 <https://wg21.link/P0226R1>`__","LWG","Mathematical Special Functions for C++17","Jacksonville","",""
+"`P0226R1 <https://wg21.link/P0226R1>`__","LWG","Mathematical Special Functions for C++17","Jacksonville","|In Progress|",""
```

Comment:
Please add a note on what is already implemented.

Reply:
I added a status page as asked for in [comment above](https://github.com/llvm/llvm-project/pull/89982#issuecomment-2080161608). Implemented in 18a320a0f523d1939262d1cfc633da6e5fae4870.

Reply:
I suppose that we do not need to duplicate that info? Should we link the status page therein though?